src/js/aos.js

```diff
@@ -38,7 +38,9 @@ let options = {
   duration: 400,
   disable: false,
   once: false,
-  startEvent: 'DOMContentLoaded'
+  startEvent: 'DOMContentLoaded',
+  root: null,
```

Comment:
Could you explain the difference between `root` and `scrollContainer` ?

Also, please add documentation in the `README.md` (https://github.com/michalsnik/aos#1-initialize-aos)